tests/sandbox/Dockerfile

```diff
@@ -1,9 +1,15 @@
-ARG BASE=containers.intersystems.com/intersystems/iris-community:2024.1
+ARG BASE=containers.intersystems.com/intersystems/irishealth-community:2024.1
 FROM ${BASE}
 
 USER root
 
 RUN apt-get update \
- && apt-get install -y jq
+ && apt-get install -y jq \
+ && apt-get install -y git
 
-USER irisowner
\ No newline at end of file
+USER irisowner
+
+RUN --mount=type=bind,src=.,dst=/home/irisowner/sandbox/ \
+  iris start iris && \
+  sh /home/irisowner/sandbox/install-artifact.sh zpm-0.7.1.xml \
```

Comment:
Just an idea, how about we substitute `zpm-0.7.1.xml` with an ARG that defaults to `zpm-0.7.1.xml`? It's more handy for cases where we want to install betas of 0.9.0